packages/client/src/Frontend/Pages/GameLandingPage_v1.tsx

```diff
@@ -79,31 +87,39 @@ export function GameLandingPage_v1() {
   const { data: walletClient } = useWalletClient();
   const {
     network: { walletClient: burnerWalletClient },
-    components: { SyncProgress },
+    components: components,
```

Comment:
```suggestion
    components,
```

Reply:
As it is now you are renaming components to the same name 🤷 not specifying any type here at all.